README.md

```diff
@@ -0,0 +1,3 @@
+# total-adapters-sample
+
+[![](https://jitpack.io/v/ArturBorowy/total-adapters-sample.svg)](https://jitpack.io/#ArturBorowy/total-adapters-sample)
```

Comment:
I guess you shouldn't use capital letters in "com.github.ArturBorowy:total-adapters-sample:Tag"

Reply:
You think it can break something, or just naming convention? It's my GitHub name in dependency, so it would be bigger change.

Reply:
I just never seen a capital letter in dependency. You can do whatever you deem right.